pkg/operator/controller/externaldns/deployment.go

```diff
@@ -38,16 +39,18 @@ import (
 )
 
 const (
-	externalDNSProviderTypeAWS      = "aws"
-	externalDNSProviderTypeGCP      = "google"
-	externalDNSProviderTypeAzure    = "azure"
-	externalDNSProviderTypeBlueCat  = "bluecat"
-	externalDNSProviderTypeInfoblox = "infoblox"
-	appNameLabel                    = "app.kubernetes.io/name"
-	appInstanceLabel                = "app.kubernetes.io/instance"
-	masterNodeRoleLabel             = "node-role.kubernetes.io/master"
-	osLabel                         = "kubernetes.io/os"
-	linuxOS                         = "linux"
+	externalDNSProviderTypeAWS          = "aws"
+	externalDNSProviderTypeGCP          = "google"
+	externalDNSProviderTypeAzure        = "azure"
+	externalDNSProviderTypeAzurePrivate = "azure-private-dns"
+	externalDNSProviderTypeBlueCat      = "bluecat"
+	externalDNSProviderTypeInfoblox     = "infoblox"
+	appNameLabel                        = "app.kubernetes.io/name"
+	appInstanceLabel                    = "app.kubernetes.io/instance"
+	masterNodeRoleLabel                 = "node-role.kubernetes.io/master"
+	osLabel                             = "kubernetes.io/os"
+	linuxOS                             = "linux"
+	privateDNSZones                     = "privatednszones"
```

Comment:
```suggestion
	azurePrivateDNSZonesResourceSubStr                     = "privatednszones"
```
Just to make it more precise, WDYT?

Reply:
Done